corehq/doctypemigrations/migrator.py

```diff
@@ -112,3 +114,22 @@ def last_seq(self):
     @property
     def cleanup_complete(self):
         return self._migration_model.cleanup_complete
+
+    def docs_are_replicating(self):
+        """Make a change in source_db and see if it appears in target_db."""
+        source_db, target_db = self.source_db, self.target_db
+        doc_id = "testing-replication-doc"
+        test_doc = {"doc_type": self.doc_types[0],
+                    "_id": doc_id,
+                    "description": self.docs_are_replicating.__doc__}
+        source_db.save_doc(test_doc)
```

Comment:
As long as the doc is deleted (ie this function doesn't fail), it works.  There's no `"_rev"` in the doc, so it'll overwrite a deleted doc with the same id.